app/src/main/java/ch/epfl/sdp/appart/user/AppUser.java

```diff
@@ -2,26 +2,30 @@
 
 import android.media.Image;
 
-import com.google.firebase.auth.FirebaseAuth;
-import com.google.firebase.auth.FirebaseUser;
-
-public class FirebaseUserAdapter implements User {
+/**
+ * This class represents a generic user of our application - the actual
+ * communication with firebase is done in the Firebase login service
+ */
+public class AppUser implements User {
     private String name;
     private String userId;
     private String email;
     private String phoneNumber;
+    private String password;
     private Image profilePicture;
```

Comment:
Following #27, I would apply also in this case the idea to store a String as image reference and let the View read the image from the database using Glide.

Reply:
Good point! done.